Framework/API/inc/MantidAPI/AnalysisDataService.h

```diff
@@ -104,17 +107,21 @@ class DLLExport AnalysisDataServiceImpl final
   //@}
 
 public:
+  enum RetrievalFlags { None = 1 << 0, UnrollGroups = 1 << 2 };
+
   /// Return the list of illegal characters as one string
   const std::string &illegalCharacters() const;
   /// Set the list of illegal characters
   void setIllegalCharacterList(const std::string &);
   /// Is the given name a valid name for an object in the ADS
   const std::string isValid(const std::string &name) const;
-  /// Overridden add member to attach the name to the workspace when a workspace
+  /// Overridden add member to attach the name to the workspace when a
```

Comment:
Nit: Are these comment changes from clang? Might be worth neatening these up so they are not spread over more lines than required

Reply:
:+1: